.github/workflows/examples_publish.yaml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - master
+      - odroid_c2
```

Comment:
Gotta remove this after it passes :)

Reply:
It passed here: https://github.com/rancher-sandbox/cOS-toolkit/runs/4314722836?check_suite_focus=true